applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -94,7 +94,9 @@ void DistanceModificationProcess::CheckDefaultsAndProcessSettings(Parameters &rP
         "check_at_each_time_step"                : true,
         "avoid_almost_empty_elements"            : true,
         "deactivate_full_negative_elements"      : true,
-        "recover_original_distance_at_each_step" : false
+        "recover_original_distance_at_each_step" : false,
+        "double_variables_list"                  : ["PRESSURE"],
+        "component_variables_list"               : ["VELOCITY_X","VELOCITY_Y","VELOCITY_Z"]
```

Comment:
I would suggest a single list to simplify usability, you can always check if the input is a double variable or a component using `KratosComponents<Variable<double>>::Has(name)` and `KratosComponents<ComponentType>::Has(name)`.

Reply:
Ok I see!